index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Weather Report</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="styles/index.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Merriweather&family=Montserrat&family=Sacramento&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <h1>Weather Report</h1>
+        <h2>For the lovely city of</h2>
+        <h2>✨</h2>
+        <h2 id="intro">Seattle</h2>
+        <h2>✨</h2>
+      </div>
+      <div class="temperature">
+        <h3>Temperature</h3>
+
+        <section>
+          <button class="minus_plus">⬇</button>
+          <span class="number" id="number">45</span>
+          <button class="plus_minus">⬆</button>
+          <button id="realTime" class="RealtimeTemperature">
+            Get Realtime Temperature
+          </button>
+        </section>
+      </div>
+      <div class="sky">
+        <h3>Sky</h3>
+        <select id="skySelect" onchange="displaySky()">
+          <option value="">Select</option>
+          <option value="☁️ ☁️ ☁️ ☀️ ☁️ ☁️">Sunny</option>
+          <option value="☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️">Cloudy</option>
+          <option value="🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧">Rainy</option>
+          <option value="🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨">Snowy</option>
+        </select>
```

Comment:
Nice dropdown! As an alternative, can you consider how you would set the event using JavaScript instead?